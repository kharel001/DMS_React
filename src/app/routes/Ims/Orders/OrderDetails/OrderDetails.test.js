import React from 'react';
import OrderDetails from './OrderDetails';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < OrderDetails {...props}  />)
    )
}

const findByTestAttr=(wrapper, val)=>{
    return wrapper.find(`[data-test='${val}']`)
}


const dummy_data=
    {name:"8",
    type:{
        image:"image",
        name:"name"
    }}


describe('OrderDetails />', () => {
  let wrapper;


  beforeEach(() => {
    wrapper = setup(dummy_data)
  });

  afterEach(() => {
    jest.clearAllMocks();
  });


it("OrderDetails snapshot", () => {
        // let wrapper = setup(dummy_data);
        expect(wrapper).toMatchSnapshot();

  
})

})
