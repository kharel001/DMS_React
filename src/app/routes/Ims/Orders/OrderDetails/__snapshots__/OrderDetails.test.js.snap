// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderDetails /> OrderDetails snapshot 1`] = `ShallowWrapper {}`;
